src/ExpressionEvaluator/Core/Source/FunctionResolver/FunctionResolver.cs

```diff
@@ -3,18 +3,17 @@
 using Microsoft.VisualStudio.Debugger;
 using Microsoft.VisualStudio.Debugger.Clr;
 using Microsoft.VisualStudio.Debugger.ComponentInterfaces;
-using Microsoft.VisualStudio.Debugger.Evaluation;
 using Microsoft.VisualStudio.Debugger.FunctionResolution;
+using Microsoft.VisualStudio.Debugger.Symbols;
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Reflection.Metadata;
-using Microsoft.VisualStudio.Debugger.Symbols;
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
     internal abstract class FunctionResolver :
```

Comment:
>FunctionResolver [](start = 28, length = 16)

This is a question on previous code: might we fold `FunctionResolver` into `FunctionResolverBase` since there is no other class deriving from this base class?